lib/tasks/data_migrations/backfill_service_upload_educator_ids.rake

```diff
@@ -0,0 +1,19 @@
+namespace :data_migration do
+  desc "Backfill all service upload IDs"
+  task backfill_service_upload_ids: :environment do
+    raise "Need an Uri ID!" unless ENV['URI_ID']
+
+    puts "Updating service upload records..."; puts;
+
+    ActiveRecord::Base.transaction do
+      ServiceUpload.all.each do |service_upload|
+        service_upload.uploaded_by_educator_id = ENV.fetch('URI_ID')
```

Comment:
Do we want to do all, or just the nil ones?

Reply:
Right now they're all `nil` and this is a one-time-only rake task so it's the same thing.